airflow/hooks/adl_hook.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from airflow.hooks.base_hook import BaseHook
+from azure.datalake.store import core, lib, multithread
+
+
+class AdlHook(BaseHook):
```

Comment:
My preference would be to call it AzureDataLakeHook instead of the abbreviates.

Reply:
@Fokko I thought about it when developing, but found others using abbreviates too, so I thought that could be a naming pattern. I'll change it and commit again.

Reply:
In this case the module would be azure_data_lake wouldn't it?

Reply:
Yes, this should be `azure_data_lake_hook.py`